scripts/globals/weaponskills.lua

```diff
@@ -547,32 +498,14 @@ xi.weaponskills.calculateRawWSDmg = function(attacker, target, wsID, tp, action,
     local ftp = xi.weaponskills.fTP(tp, wsParams.ftp100, wsParams.ftp200, wsParams.ftp300) + calcParams.bonusfTP
 
     -- Calculate critrates
-    local critrate = calculateDEXvsAGICritRate(attacker, target)
+    local criticalRate = 0
```

Comment:
btw setting it out here wasn't giving crit rate to non-crit ws because each swing checked `wsParams.canCrit` then math.random, but def cleaner to just do everything inside the block below

Reply:
This is why weaponskills need a nuclear rewrite from scratch